SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/engine/model/ConversationAtom.java

```diff
@@ -29,7 +28,7 @@ public String getTag() {
         return tag;
     }
 
-    public AtomStyle getStyle(){
+    public ConversationStyle getStyle(){
```

Comment:
The ConversationStyle object is an object that can be the style of a button, normal atom such as text/image/etc, or a page styling.